packages/terra-form-field/src/Field.jsx

```diff
@@ -100,11 +106,15 @@ const Field = (props, { intl }) => {
     isLabelHidden,
     label,
     labelAttrs,
+    maxWidth,
     required,
     showOptional,
+    style,
     ...customProps
   } = props;
 
+  const customStyles = Object.assign({ maxWidth }, style);
```

Comment:
Let's update this to:
```suggestion
  const customStyles = maxWidth ? Object.assign({ maxWidth }, style) : style;
```

This will remove the following from the jest snapshots:
```
style={
    Object {
      "maxWidth": undefined,
    }
  }
```